lib/code_corps_web/controllers/project_category_controller.ex

```diff
@@ -11,6 +11,9 @@ defmodule CodeCorps.ProjectCategoryController do
   plug :load_and_authorize_resource, model: ProjectCategory, only: [:delete]
   plug JaResource
 
+  @spec model :: module
+  def model, do: CodeCorps.ProjectCategory
```

Comment:
Good question. Was gonna comment on that myself to elaborate.

We use the `ja_resource` package in our controllers, to auto-load resources. 

It's been getting less and less useful for us, though, and this is just another case where more pain and no benefit is added.

`ja_resource` auto loads by assuming an `id` in the params. That part works in most of our cases, except in the case where we use slugs.

It also assumes/infers the model name from the controller name, so in this specific case, it will assume 

`CodeCorpsWeb.ProjectCategoryController` -> `CodeCorpsWeb.ProjectCategory`.

When the only namespace was `CodeCorps`, this worked. Now, it does not anymore, so we have to override the `model` function provided to us by `ja_resource` to get it working again.

With this one more thing we have to write in our controller and the fact that phoenix 1.3 wants to get away from models and into contexts, as well as the addition of the `FallbackController`, plus several other problems `ja_resource` is creating for us, I'm going to recommend we ditch the package completely.